runtime/gc_vlhgc/CopyScanCacheVLHGC.hpp

```diff
@@ -74,7 +74,7 @@ class MM_CopyScanCacheVLHGC : public MM_CopyScanCache
 	 */	
 	MMINLINE bool isSplitArray() const
 	{
-		return (J9VM_MODRON_SCAVENGER_CACHE_TYPE_SPLIT_ARRAY == (flags & J9VM_MODRON_SCAVENGER_CACHE_TYPE_SPLIT_ARRAY));
+		return (OMR_COPYSCAN_CACHE_TYPE_SPLIT_ARRAY == (flags & OMR_COPYSCAN_CACHE_TYPE_SPLIT_ARRAY));
```

Comment:
Please use `OMR_ARE_ALL_BITS_SET()` and remove the unwanted outermost parentheses:
```c
  return OMR_ARE_ALL_BITS_SET(flags, OMR_COPYSCAN_CACHE_TYPE_SPLIT_ARRAY);
```